components/ContentTemplates/ImagesTemplate.tsx

```diff
@@ -406,9 +407,10 @@ export const ImagesTemplate = () => {
 					</a>
 				</TemplateSection>
 			</article>
-			<p className="pageLastUpdated">
+			{/* <p className="pageLastUpdated">
```

Comment:
Please remove the code instead of commenting it out.

Reply:
Ok